build/release/release.sh

```diff
@@ -44,18 +44,26 @@ SYS=${LNDBUILDSYS:-"
         freebsd-386
         freebsd-amd64
         freebsd-arm
+        illumos-amd64
         linux-386
         linux-amd64
         linux-armv6
         linux-armv7
         linux-arm64
         linux-ppc64
+        linux-ppc64le
+        linux-mips
+        linux-mipsle
         linux-mips64
         linux-mips64le
+        linux-s390x
         netbsd-386
         netbsd-amd64
+        netbsd-arm
         openbsd-386
         openbsd-amd64
+        openbsd-arm
```

Comment:
Any reason for adding some of these and not others? Looking at `go tool dist list`, there are still some missing.

Reply:
> `go tool dist list`

TIL! Added in some missing `arm` targets. `plan9` is still experimental and `nacl` will be sunset after go1.13, so those are omitted. Feel free to suggest removing any of these, was just trying to lay out all the options.